src/commands/adirs.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../lib/command';
+import { makeEmbed, makeLines } from '../lib/embed';
+import { CommandCategory } from '../constants';
+
+const ADIRS_IMAGE_URL = 'https://media.discordapp.net/attachments/785976111875751956/818095298538504272/image0.png';
+
+export const adirs: CommandDefinition = {
+    name: ['ADIRS', 'adirs'],
+    description: 'Display help with ADIRS alignment',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | ADIRS align',
+        description: makeLines([
+            'On the overhead panel you will see the three switches under 'ADIRS'. Turn these three to the 'NAV' position. It takes several minutes for the ADIRUs to align.',
```

Comment:
Single quotes in strings need to be escaped. Which version of Node are you using? My version (v14.16.0) does not allow me to run this code.

```suggestion
            'On the overhead panel you will see the three switches under \'ADIRS\'. Turn these three to the \'NAV\' position. It takes several minutes for the ADIRUs to align.',
```